regolith/helpers/makeappointmentshelper.py

```diff
@@ -106,11 +107,29 @@ def sout(self):
                                 f"person: {[p].get('_id')} appointment: {a.get('_id')} grant: {g.get('_id')} "
                                 f"date: {str(day)}")
 
+        for g in self.gtx["grants"]:
+            end_date = get_dates(g)['end_date']
+            if get_grant_amount(g, end_date) > 100:
+                # what is the required 'underspend' amount?
+                underspent.append(g.get('_id'))
+            elif get_grant_amount(g, end_date) < 100:
+                # what is the required 'overspend' amount?
+                overspent.append(g.get('_id'))
+
+
 
 
         if outdated:
             print("Appointments supported on outdated grants:")
             for appt in outdated:
                 print(appt)
+        if underspent:
+            print("Underspent grants:")
+            for g in underspent:
+                print(g)
```

Comment:
I think it would be helpful to have the amount of the over/underspend (in student/pd - months)

Reply:
done